services/hanke-service/build.gradle.kts

```diff
@@ -152,6 +156,13 @@ tasks {
 			exceptionFormat = TestExceptionFormat.FULL
 		}
 	}
+
+	jacocoTestReport {
+		executionData.setFrom(
+            fileTree(buildDir).include("/jacoco/test.exec", "/jacoco/integrationTest.exec")
+        )
+        dependsOn("integrationTest")
```

Comment:
I'm not sure if this task should depend on the integration test task. This will cause the integration tests to be run, even if you just want to create the report from the test run you just finished.

At least, if it depends on this task, it should also depend on the test task.

Reply:
It is possible to run them separately. The fact still is that there is a dependency from report to the tests. I.e. when you run jacocoTestReport, the tests must have been run before to have data for coverage.

How about this: 		`shouldRunAfter("test", "integrationTest")` ?

And then we should have some instructions in readme.

Reply:
shouldRunAfter
> For each supplied task, this action adds a task 'ordering', and does not specify a 'dependency' between the tasks. As such, it is still possible to execute 'taskY' without first executing the 'taskX' in the example.

This will not specify an actual dependency, but it will ensure at least correct ordering if multiple tasks are executed.

Reply:
``` 
./gradlew test integrationTest jacocoTestReport                                                                                                                
> Task :services:hanke-service:integrationTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':services:hanke-service:integrationTest' (type 'Test').
  - Gradle detected a problem with the following location: '/haitaton-backend/services/hanke-service/build/jacoco/integrationTest.exec'.
    
    Reason: Task ':services:hanke-service:jacocoTestReport' uses this output of task ':services:hanke-service:integrationTest' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
```

I'm getting this error if I try to remove the declared dependency.

Reply:
Looks like I didn't try to run them at the same time, except with the check task I made. Seems like it's trying to run the report at the same time as it's running the integration tests. Adding the `shouldRunAfter` didn't help with this, since it allows the tasks to be run in parallel. The stronger `mustRunAfter` did.

Reply:
It is now using mustRunAfter